### PR TITLE
Desktop version Portfolio

### PR DESCRIPTION
**Pull request by Itswali**
This includes HTML and CSS file

- I added HTML file for both mobile and desktop version of portfolio.
- I add CSS file and use media queries for desktop on same HTML file.
- I use linters to identify and remove any errors.
- I used gitignore file to filter unwanted files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,298 +9,327 @@
...
                         <li><a href="#"><img class="okk" src="./Images/header/ok.png" alt="ok"></a></li>
-
                     </ul>
-
-
-
                 </div>
-
             </div>
         </div>
```
Comment (ridwanediallo) [inline index.html]: - [ ] I would kindly recommend making sure that your portfolio looks like the Figma template. Your portfolio should look good from the breakpoint `768` to `1440`. Your hero section is not displaying as expected. On desktop view, there is no background image and the text and the icons moved to the left. Please make sure to fix that.

## Your hero section 👇️
Screenshot from 2022-09-29 21-35-53

## Figma design 👇️

Screenshot from 2022-09-29 22-18-31

File: index.html [hunk 2]
```diff
@@ -10,298 +9,327 @@
...
+                <img src="./Images/header/desktop4.png" alt="work_tonic">
+            </div>
         </div>
     </section>
-
     <section class="detail_section">
-
         <div id="about">
             <div id="box1">
                 <h2>About Myself</h2>
```
Comment (ridwanediallo) [inline index.html:335]: - [ ] Kindly make sure to fix your about me section to stick to the Figma design.

## Your design 👇️
Screenshot from 2022-09-29 21-38-09

## The one on Figma 👇️
Screenshot from 2022-09-29 22-47-51

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @Itswali , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly your mobile version should still look as expected. So make to fix all your work sections and the about me section. 

## One of your work section  👇️
Screenshot from 2022-09-29 21-42-42

## The design on Figma 👇️

Screenshot from 2022-09-24 00-29-46

## Your about me section 👇️
Screenshot from 2022-09-29 21-43-29

The about me on Figma
Screenshot from 2022-09-29 23-02-11

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Itswali** [comment]
Improved files and documented properly as suggested.

File: index.html [hunk 3]
```diff
@@ -10,298 +9,327 @@
...
                         <h1>I'm Wali Glad to see You!</h1>
-
-                    </div>
-
+                    </div> <br>
                     <div class="my_description">
                         <p>I’m a software developer! I can help you build a product , feature or website Look through
-                            some
-                            of my work and experience! If you like what you see and have a project you need coded, don’t
+                            some of my work and experience! If you like what you see and have a project you need coded,
```
Comment (nedjwab) [inline index.html:37]: [ ] kindly Fix the `font-weight` of this paragraph to match the Figma design.🤓
Your design : 
paragraph

Figma design : 
designtext

File: index.html [hunk 4]
```diff
@@ -10,298 +9,327 @@
...
-                    <img src="./Images/header/Counter.png" alt="dot">
-
-                    <p>Back End Dev</p>
-
-                    <img src="./Images/header/Counter.png" alt="dott">
-                    <p>2015</p>
-
-
-
+            <div class="for_texts">
```
Comment (nedjwab) [inline index.html:62]: [ ] kindly fix the card design to match the template. 🤓
Your cards : 
card1
deskcard2
descard


The Figma design :
designcard

File: index.html [hunk 5]
```diff
@@ -10,298 +9,327 @@
...
+                <img src="./Images/header/desktop4.png" alt="work_tonic">
+            </div>
         </div>
     </section>
-
     <section class="detail_section">
-
         <div id="about">
             <div id="box1">
                 <h2>About Myself</h2>
```
Comment (nedjwab) [inline index.html:335]: [ ] kindly use the same font-weight for the languages in the about me section to match the design.🤓
Your design : 
deskicons

Figma design : 
designicons

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Itswali 

Nice job✔, I liked your implementation 🔥, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing  the project requirements 🔥🔥, but you still have some issues:  
[ ] kindly fix your mobile version as the previous reviewer has mentioned :
mobile1
mobile2



Check the comments below 👇
  
## [Optional] suggestions:

-  Nothing to mention. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Itswali** [comment]
fix the desktop version as suggested.    Last review<3

File: index.html (hunk 4 above)
Comment (Alfredbis29) [reply index.html:62]: - [ ] I can see that you made some changes but your design still need to be  improved as required in the previous review :upside_down_face: 
This is how your cards still looking like 
frappesection

Figma Design 
designcard

File: index.html (hunk 5 above)
Comment (Alfredbis29) [reply index.html:335]: - [ ] Great job so far but there still something to work on for your project to match the figma Design as required in the previous review :upside_down_face: , I noticed that after making changes your languages image are not displaying , you should fix the path of these icons and make sure that your following the figma design.
This is how your design looks like after changes :point_down: 

frape3media

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @Itswali ,
 
 Great work on making the changes requested by a previous reviewer clap You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
 
# STATUS: REQUEST CHANGES 
 soclose

 ## Highlights
 * [x]  All linters are green
 * [x]  Descriptive PR `summary`
 * [x]  The project is documented in a professional way
 
 ## Required Changes 
 _Check the comments under the review._
 
 ## Optional suggestions
 * [x]  You may also consider adjusting your `typo` (`font-size`) to match the ones provided in Figma
 
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**